src/libs/antares/writer/private/zip_writer.hxx

```diff
@@ -11,8 +12,9 @@ void ZipWriter::addEntryFromBufferHelper(const std::string& entryPath, ContentTy
         return;
 
     EnsureQueueStartedIfNeeded ensureQueue(this, pQueueService);
-    pQueueService->add(
-      new ZipWriteJob<ContentType>(*this, entryPath, entryContent, pDurationCollector),
-      Yuni::Job::priorityLow);
+    Antares::Concurrency::AddTask(*pQueueService,
+                                  ZipWriteJob<ContentType>(*this, entryPath, entryContent, pDurationCollector),
+                                  Yuni::Job::priorityLow);
```

Comment:
Unfortunately, the result of this is not stored in a `FutureSet`, so we won't be able to catch exceptions.

Maybe we could wrep `QueueService` and `FutureSet` together and expose a simpler `add` method ?

```cpp
class Wrapper {
Yuni::QueueService* queue;
FutureSet results;
public:
add(...);
};
```
Note that `Wrapper` may be returned by `createThreadPool` that is defined in the tests, instead of a mere QueueService.